twitch_bot/cogs/twitch.py

```diff
@@ -72,10 +72,10 @@ def __init__(self, silent_initial=True, interactions=False):
 
         self.silent_initial = silent_initial
 
-    async def sh_person(self, author, channel):
+    async def sh_person(self, author, channel: Channel):
```

Comment:
se for fortificar a tipagem, pode fazer com `author` também.

Reply:
Posso fazer isso sim, vai trazer uma maior clareza pro código.